api/client/src/main/java/org/projectnessie/client/auth/oauth2/TokensResponseBase.java

```diff
@@ -83,9 +95,9 @@ default RefreshToken getRefreshToken() {
   @Nullable
   @JsonProperty("expires_in")
   @JsonUnwrapped
-  @JsonSerialize(using = JacksonSerializers.InstantToSecondsSerializer.class)
-  @JsonDeserialize(using = JacksonSerializers.SecondsToInstantDeserializer.class)
-  Instant getAccessTokenExpirationTime();
+  @JsonSerialize(using = JacksonSerializers.DurationToSecondsSerializer.class)
+  @JsonDeserialize(using = JacksonSerializers.SecondsToDurationDeserializer.class)
+  Duration getAccessTokenExpiresIn();
```

Comment:
I wonder whether the (de)serialization to a `Duration` type is really necessary. The OAuth & Keycloak specs say that this is an integer value representing 0 or or more seconds.
I suspect, just using `Integer` instead of `Duration` here and below is probably easier overall.

The function names could be extended with `...Seconds()` tho.

Reply:
Just using an `Integer` also saves a bunch of code.

Reply:
Note that we lose the ability to use sub-second durations in unit tests – but all in all, I think that's fine.